### PR TITLE
LPS-193126

### PR DESCRIPTION
**Pull request by icgsbr**
# Jira Link
- [LPS-193126](https://liferay.atlassian.net/browse/LPS-193126)

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormPortlet.java [hunk 1]
```diff
@@ -218,17 +222,24 @@ protected void setRenderRequestAttributes(
 			_ddmFormInstanceRecordVersionLocalService, _ddmFormInstanceService,
 			_ddmFormInstanceVersionLocalService, _ddmFormRenderer,
 			_ddmFormValuesFactory, _ddmFormValuesMerger,
-			_ddmFormWebConfigurationActivator.getDDMFormWebConfiguration(),
-			_ddmStorageAdapterRegistry, _groupLocalService, _jsonFactory,
-			_npmResolver, _objectFieldLocalService,
-			_objectFieldSettingLocalService, _objectRelationshipLocalService,
-			_portal, renderRequest, renderResponse, _roleLocalService,
-			_userLocalService, _workflowDefinitionLinkLocalService);
+			_getDDMFormWebConfiguration(), _ddmStorageAdapterRegistry,
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormPortlet.java:225]: @icgsbr please inline, no need to create a new method that is only used once

```
...
ConfigurationProviderUtil.getCompanyConfiguration(
    DDMFormWebConfiguration.class, 
    CompanyThreadLocal.getCompanyId()), 
_ddmStorageAdapterRegistry, _groupLocalService
...
```

Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormPortlet.java:225]: @icgsbr could you please delete the `DDMFormWebConfigurationActivator _ddmFormWebConfigurationActivator` variable as it is no longer being used?? Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by icgsbr** [comment]
ci:test:stable

**Comment by icgsbr** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:test:stable

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**